api.py

```diff
@@ -52,13 +54,32 @@ def get(self, token, url):
 
         return (response.status_code, response.json()['response'])
 
-    # TODO: Figure out correct header format
-    """ Make authenticated PUT request for uploading PDF """
-    def put(self, token, url):
+    # can't use this because POST requests not accepted
+    def upload_pdf_wrong(self, token, url, filename):
```

Comment:
I wouldn't have committed this.

You can use `git add -p` to choose what to add and what not to add.

It is a good idea to use this anyway so that you can verify that you are only committing things that you want to be.